tools/booster/Proxy.hs

```diff
@@ -197,6 +217,7 @@ respondEither mbStatsVar booster kore req = case req of
                                     ( currentDepth + boosterResult.depth + koreResult.depth
                                     , time + bTime + kTime
                                     , koreTime + kTime
+                                    , maybe rpcLogs (rpcLogs ++) (liftA2 (++) boosterResult.logs koreResult.logs)
```

Comment:
We must pass the traces from previous iterations and return them in the final `ExecuteResponse`